SU-HSDO/suhumsci
# SHS:5892: Site managers can perform actions on user accounts that don't make sense with SSO

# READY FOR REVIEW

## Summary
- Remove "Cancel the selected user account(s)" and "Update URL Alias" bulk actions from the user list view
- In the user edit form, hide the URL alias related fields for all users and the "Cancel User" button for non admin users

## Need Review By (Date)
12/04

## Urgency
medium

## Steps to Test
1. Log into the [tugboat test site](https://hs-colorful-kea8cw1nuentjq5kebwi6lryjgqsr8au.tugboatqa.com/user/login)
2. Visit the [People view](https://hs-colorful-kea8cw1nuentjq5kebwi6lryjgqsr8au.tugboatqa.com/admin/users)
3. Select one or more users. Confirm one or more users and confirm that the following actions are not available anymore on the "Action" selector that appears at the bottom of the list:
    - Update URL alias
    - Cancel the selected user account(s)
4. For any user, click on the "Edit" button in the "Operations" column at the right. Confirm that:
    - There are no fields related to the URL alias visible
    - The "Cancel account" button is present at the bottom of the form
5. [Masquerade](https://hs-colorful-kea8cw1nuentjq5kebwi6lryjgqsr8au.tugboatqa.com/admin/users/masquerade) as an user with the "Site Manager" role (Lindsey)
6. Repeat steps 2 through 4. Everything should be the same as for the administrator user, **except that the "Cancel account" button in the user edit form shouldn't be present**.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208827336999823
